solver/src/settlement.rs

```diff
@@ -205,98 +258,117 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::just_underscores_and_digits)]
     fn test_buy_order_surplus() {
         // Two goods are worth the same (100 each). If we were willing to pay up to 60 to receive 50,
         // but ended paying the price (1) we have a surplus of 10 sell units, so a total surplus of 1000.
+
+        // I hope there is a better way :)
+        let _5000 = BigRational::from_u32(5000).unwrap();
```

Comment:
I don't think so because BigRational does not implement From<u32> and so on directly. I would use a small helper functions (or macro) here to avoid the repetition. Something like 

```rust
        fn r(u: u128) -> BigRational {
            BigRational::from_u128(u).unwrap()
        }

        let _5000 = r(5000);
```

and at that point you can put that directly in the assert instead of going through a variable.

Reply:
Thanks! It's here a2a4e.